nightwatch.conf.js

```diff
@@ -0,0 +1,50 @@
+const seleniumServer = require('selenium-server');
+const phantomjs = require('phantomjs-prebuilt');
```

Comment:
should we still use the deprecated `phantomjs`? I don't even see it in the Nightwatch documentation: http://nightwatchjs.org/gettingstarted#browser-drivers-setup maybe that now we have Headless and Firefox headless modes? 👍

Reply:
Great point! I'll look at that.